### ISSUE TITLE
[rush] postRushInstall script working directory

### ISSUE THREAD

**Issue by GongT** [microsoft/web-build-tools#1352]
This event hook always run in current working directory.

Eg:
`postRushInstall` set to `node common/scripts/post-install.js`

Run `rush --debug update` in `library` directory in project root
```
The command failed with exit code 1
internal/modules/cjs/loader.js:628
    throw err;
    ^

Error: Cannot find module '/PATH_TO_PROJECT/library/common/scripts/post-install.js'
```

Run `rush update` at project root, it works well.

**Comment by iclanton** [comment]
I've seen this issue too. I think this feature needs some revision.

**Comment by iclanton** [comment]
This has been fixed in Rush 5.10.3.
